conn/conn.go

```diff
@@ -78,6 +90,10 @@ var (
 )
 
 func (fn ReceiveFunc) PrettyName() string {
+	return prettyName(fn)
+}
+
+func prettyName(fn interface{}) string {
```

Comment:
What's the point of this indirection? I don't see prettyName used anywhere, but only PrettyName.

Reply:
Ah, this is vestigial, there was an iteration that had two versions of this function (called by two variants of receivefn, one did batches, the other non-batches) that was later refactored out. I'll flatten it down.